src/string/immutable.zig

```diff
@@ -968,35 +968,17 @@ pub fn eqlComptimeCheckLenWithType(comptime Type: type, a: []const Type, comptim
     return eqlComptimeCheckLenWithKnownType(comptime Type, a, if (@typeInfo(@TypeOf(b)) != .pointer) &b else b, comptime check_len);
 }
 
-pub fn eqlCaseInsensitiveASCIIIgnoreLength(
-    a: string,
-    b: string,
-) bool {
-    return eqlCaseInsensitiveASCII(a, b, false);
-}
-
-pub fn eqlCaseInsensitiveASCIIICheckLength(
-    a: string,
-    b: string,
-) bool {
-    return eqlCaseInsensitiveASCII(a, b, true);
-}
-
-pub fn eqlCaseInsensitiveASCII(a: string, b: string, comptime check_len: bool) bool {
-    if (comptime check_len) {
-        if (a.len != b.len) return false;
-        if (a.len == 0) return true;
-    }
-
-    bun.unsafeAssert(b.len > 0);
-    bun.unsafeAssert(a.len > 0);
-
-    return bun.c.strncasecmp(a.ptr, b.ptr, a.len) == 0;
+/// Case insensitive equality check for ASCII strings.
+///
+/// Note that this function performs bound checks, but is marked as inline, so the caller may also
+/// perform bounds checks without a performance penalty.
```

Comment:
Does this comment apply considering `std.ascii.eqlIgnoreCase` is not marked inline?